inference-engine/ie_bridges/java/cpp/tensor_desc.cpp

```diff
@@ -0,0 +1,201 @@
+#include <inference_engine.hpp>
+
+#include "openvino_java.hpp"
+#include "enum_mapping.hpp"
+#include "jni_common.hpp"
+
+using namespace InferenceEngine;
+
+JNIEXPORT jlong JNICALL Java_org_intel_openvino_TensorDesc_GetTensorDesc(JNIEnv *env, jobject obj, jint precision, jintArray dims, jint layout)
+{
+    static const char method_name[] = "GetTensorDesc";
+    try
+    {
+        auto n_precision = precision_map.find(precision_e(precision));
+
+        if (n_precision == precision_map.end())
+            throw std::runtime_error("No such precision value!");
+
+        auto n_layout = layout_map.find(layout_e(layout));
+
+        if (n_layout == layout_map.end())
+            throw std::runtime_error("No such layout value!");
+
+        const jsize length = env->GetArrayLength(dims);
+        jint *data = env->GetIntArrayElements(dims, 0);
+
+        std::vector<size_t> sVector;
+        
+        for (int i = 0; i < length; i++)
+            sVector.push_back(static_cast<size_t>(data[i]));
```

Comment:
Please check if the following code works:
```cpp
std::vector<size_t> sVector(data.begin(), data.end());
```

Reply:
```error: request for member ‘begin’ in ‘data’, which is of non-class type ‘jint* {aka int*}
error: request for member ‘end’ in ‘data’, which is of non-class type ‘jint* {aka int*}```

Reply:
Maybe ` std::vector<size_t> sVector(data, data + length);`?